runtime/plugin/lsp.vim

```diff
@@ -0,0 +1,28 @@
+" TODO(tjdevries): Move this to autoload?
+
+execute('luafile ' . expand('<sfile>:h') . '/../lua/lsp/plugin.lua')
```

Comment:
Should be something like `lua lsp = require('lsp.plugin').setup()`. (Needs #6789 or you adding a path to `package.path`.)